osf/metrics/reporters/spam_count.py

```diff
@@ -0,0 +1,84 @@
+from osf.models import OSFUser
+
+from osf.metrics.reports import SpamSummaryReport
+from ._base import MonthlyReporter
+from osf.models import PreprintLog, NodeLog
+from osf.models.spam import SpamStatus
+
+
+class SpamCountReporter(MonthlyReporter):
+
+    def report(self, report_yearmonth):
+        target_month = report_yearmonth.target_month()
+        next_month = report_yearmonth.next_month()
+
+        report = SpamSummaryReport(
+            report_yearmonth=str(report_yearmonth),
+            # Node Log entries
+            confirmed_spam_node=NodeLog.objects.filter(
```

Comment:
I think this no longer matches what's in the report definition. Go through and make sure all of these agree, please